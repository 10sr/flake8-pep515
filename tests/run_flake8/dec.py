a = 123_456_789
b = 1234_5678
