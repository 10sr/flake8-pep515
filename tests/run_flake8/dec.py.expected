./dec.py:2:14: NSP001 DEC Invalid
